tests/integration/unit_test/test_unit_test_dotnet6.py

```diff
@@ -18,4 +18,7 @@ class UnitTest_dotnet6_cookiecutter_aws_sam_hello_step_functions_sample_app(Unit
     code_directories = ["tests/StockBuyer.Test", "tests/StockChecker.Test", "tests/StockSeller.Test"]
 
 
+class UnitTest_dotnet6_cookiecutter_aws_sam_quickstart_sns_dotnet_sample_app(UnitTestBase.DotNetCoreUnitTestBase):
+    directory = "dotnet6/cookiecutter-aws-sam-quickstart-sns-dotnet"
+    code_directories = ["test/{{cookiecutter.project_name}}.Tests"]
```

Comment:
@neville-lewis change here:
from:
["test/{{cookiecutter.project_name}}.Tests"]
to:
["test/project.Tests"]